src/Facebook/InstantArticles/Client/Helper.php

```diff
@@ -37,6 +37,7 @@ public function __construct($facebook)
      * @param string $appID
      * @param string $appSecret
      *
+     * @return static
```

Comment:
Guess this is more <type> <what is inside> standard.
Can you add this, please?
`@return Helper new Instance.`